packages/components/src/dashicon/README.md

```diff
@@ -13,3 +21,372 @@ const MyDashicon = () => (
 	</div>
 );
 ```
+
+## Props
+
+The component accepts the following props:
+
+### icon
+
+The name of the icon. See the list below for all possible values.
+
+-   Type: `string`
+-   Required: Yes
+
+### className
+
+Additional classes for the `<span>` element.
+
+-   Type: `string`
+-   Required: No
+
+## List of available icons
```

Comment:
As a consequence of what explained in the previous comment, we should remove this section.

I believe that it's a better idea to link out to the existing official docs — this will remove the need for maintaining an up-to-date list in this README.